trie/trie_test.go

```diff
@@ -546,16 +546,3 @@ func updateString(trie *Trie, k, v string) {
 func deleteString(trie *Trie, k string) {
 	trie.Delete([]byte(k))
 }
-
-func TestDecodeNode(t *testing.T) {
-	t.Parallel()
-	var (
-		hash  = make([]byte, 20)
-		elems = make([]byte, 20)
-	)
-	for i := 0; i < 5000000; i++ {
-		rand.Read(hash)
-		rand.Read(elems)
-		decodeNode(hash, elems)
-	}
-}
```

Comment:
Why delete this one?

Reply:
this test seems useless, and it tests nothing, so i decide to delete it. already rollback.